src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/HmppsRestrictedPatientsApiExceptionHandler.kt

```diff
@@ -78,6 +93,7 @@ class HmppsRestrictedPatientsApiExceptionHandler {
       .body(
         ErrorResponse(
           status = INTERNAL_SERVER_ERROR.value(),
+          errorCode = "UPSTREAM_ERROR",
```

Comment:
Differentiates errors from failing upstream services with 500s caused by a bug in our service. Helps the client display nicer error messages to end users (e.g. whether to retry later).